app/controllers/event_members_controller.rb

```diff
@@ -0,0 +1,38 @@
+class EventMembersController < ApplicationController
+  before_action :set_event, :set_guild, :set_game
+  def new
+    @event_member = EventMember.new
+  end
+
+  def create
+    @event_member = EventMember.new(event_member_params)
+    if @event_member.save
+      redirect_to @guild.find(params[:id])
+    else
+      root_path
+    end
+  end
+
+  def destroy
+    @event_member.destroy
+    redirect_back(fallback_location: root_path)
```

Comment:
I think this fallback should be the guild path

Reply:
changes done